frontend/src/components/ModeToggle.tsx

```diff
@@ -26,14 +27,14 @@ function ModeToggle({ className }: { className?: string }) {
   return (
     <div className={cn('flex items-center space-x-2', className)}>
       <span className="text-sm font-medium text-gray-900 dark:text-gray-100">
-        <FontAwesomeIconWrapper icon="fa-regular fa-lightbulb" />
+        <FontAwesomeIcon icon={faSun} />
```

Comment:
Could you update this to have only one icon and show relevant based on theme? Same we have it right now it [staging](https://nest.owasp.dev/projects/contribute/)?
```suggestion
        <FontAwesomeIcon icon={dark ? faMoon : faSun} />
```

Reply:
Okay I understood,  we need either moon or the sun icon, sure ill implement this.